tests/unittests/tests-rtc/tests-rtc.c

```diff
@@ -224,6 +224,35 @@ static void test_rtc_mktime(void)
     TEST_ASSERT_EQUAL_INT(98197931, rtc_mktime(&t));
 }
 
+static void test_mktime(void)
```

Comment:
Nit:
```suggestion
static void test_mktime_is_available(void)
```

I'm a bit confused over your goal for this test. I assume you want to know if mktime is available and "works" (as in: it's not stub). Is that correct?

Reply:
i want to ensure mktime is working the same on our devices  aka the 1st second for 1970 returns 1 (unix-time) like many people and pkgs assume

Reply:
The test will currently fail on native if not running with TZ=UTC  or something similar

since nativ uses glibc which will uses localtime to convert the time stamp